nixos/modules/services/network-filesystems/glusterfs.nix

```diff
@@ -30,6 +46,39 @@ in
         description = "Extra flags passed to the GlusterFS daemon";
         default = [];
       };
+
+      tlsSettings = mkOption {
+        description = ''
+          Make the server communicat via TLS.
```

Comment:
typo `communicate`

Reply:
I recommend adding a link to the docs: https://gluster.readthedocs.io/en/latest/Administrator%20Guide/SSL/